### PR TITLE
chore(ci): Simplify Slack part

### PR DESCRIPTION
**Pull request by janpio**
This quickly simplifies the (repeated) Slack jobs in 2 workflows.

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -30,19 +32,9 @@ jobs:
     if: success() && (github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/integration' || github.ref == 'refs/heads/patch-dev' || github.ref == 'refs/heads/latest')
     steps:
       - uses: actions/checkout@v2
-
-      - name: Install Dependencies
-        run: yarn install
```
Comment (janpio) [inline .github/workflows/test.yaml:35]: This is not needed, the shell script installs its own dependencies

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -30,19 +32,9 @@ jobs:
...
-
       - uses: actions/setup-node@v2
-        with:
-          node-version: 12
```
Comment (janpio) [inline .github/workflows/test.yaml:39]: No dependency to Node 12, should work with stable all the time

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -30,19 +32,9 @@ jobs:
...
-
       - name: report success to slack
         run: bash .github/scripts/slack-workflow-status.sh ":white_check_mark:"
-        env:
-          CI: 1
-          SLACK_WEBHOOK_URL_WORKFLOWS: ${{ secrets.SLACK_WEBHOOK_URL_WORKFLOWS }}
```
Comment (janpio) [inline .github/workflows/test.yaml:45]: Not necessary here as defined globally

**Review by Jolg42 — APPROVED** [review]
Nice and clean! 🧹

**Pull request merged**